.circleci/config.yml

```diff
@@ -288,6 +289,11 @@ workflows:
           image: fv3fit
           requires:
             - pytest
+      - build_default:
+          name: "build/push fv3fit_torch image"
```

Comment:
What's the difference between the `fv3fit_torch` image and the `fv3fit` image? We already build a `prognostic_run_gpu` image... was it not possible to use this for your training?

Reply:
Or alternatively, could we bump the tag for the `nvidia/cuda` base of the `prognostic_run_gpu` image to the version you need? Does this cause issues for @frodre?

Reply:
We exclusively use the the `fv3fit` image now for tensorflow training on GPUs, which is the reason for the separation (tf is not compatible with CUDA that works with pytorch 😟).  

We held the prog_run image in case we were going to try prognostic runs with a gpu, but we haven't used it in a while.  Could probably remove that image from the CI stack.

Reply:
I see. Well yeah we should just be mindful of the creep of more and more images. I would support not building the prognostic run gpu image if its unused.

Reply:
Yes I also wanted to avoid adding a new image, unfortunately no cuda version works for both.

Reply:
I don't have much knowledge about running the prognostic run on GPUs, but could that be an efficient way to parallelize the inference using reservoirs over many smaller subdomains @mcgibbon?

Reply:
I'm not sure what you mean, we can chat about it on slack.